In test/parser/let.error.2.pytch, line 1, character 4:
Expected pattern[1004]: I was expecting a pattern after 'let'.
   +-------------------------------------------------+
   | Error: I was expecting a pattern after 'let'.   |
   +-------------------------------------------------+
   | test/parser/let.error.2.pytch                   |
 1 | let                                             |
   |    ^                                            |
 2 | let foo = bar                                   |
   +-------------------------------------------------+
   | Note: This is the beginning of the let-binding. |
   +-------------------------------------------------+
   | test/parser/let.error.2.pytch                   |
 1 | let                                             |
   | ^~~                                             |
 2 | let foo = bar                                   |
   +-------------------------------------------------+
In test/parser/let.error.2.pytch, line 3, character 4:
Expected pattern[1004]: I was expecting a pattern after 'let'.
   +-------------------------------------------------+
   | Error: I was expecting a pattern after 'let'.   |
   +-------------------------------------------------+
   | test/parser/let.error.2.pytch                   |
 2 | let foo = bar                                   |
 3 | let                                             |
   |    ^                                            |
   +-------------------------------------------------+
   | Note: This is the beginning of the let-binding. |
   +-------------------------------------------------+
   | test/parser/let.error.2.pytch                   |
 2 | let foo = bar                                   |
 3 | let                                             |
   | ^~~                                             |
   +-------------------------------------------------+
